group.py

```diff
@@ -2,4 +2,53 @@
 
 # Your code to go here...
 
-my_group =
+from statistics import mean
+
+my_group = {
+        "Jill": { "age": 26,
+                "jobs": ["Biologist"],
+                "connections": {
+                    "friend": ["Zalika"],
+                    "partner": ["John"]
+                    }
+                },
+            "Zalika": {
+                "age": 28,
+                "jobs": ["Artist"],
+                "connections": {
+                    "friend": ["Jill"]
+                    }
+            },
+            
+            "John": {
+                "age": 27,
+                "jobs": ["Writer"],
+                "connections": {
+                    "partner": ["Jill"]
+                    }
+            },
+            "Nash": {
+                "age": 34,
+                "jobs": ["Chef"],
+                "connections": {
+                    "cousin": ["John"],
+                    "landlord": ["Zalika"]   
+                }
+           }
+}
+
+
+def max_age(group):
+    return max([person["age"] for person in list(group.values())])
```

Comment:
FYI, Python dict.values() returns a generator which means in a list comprehension you don't need to convert it to a list because it knows how to iterate through it
```suggestion
    return max([person["age"] for person in group.values()])
```